eureka2-integration/src/test/java/com/netflix/eureka2/integration/startup/FileBasedServerStartupAndShutdownIntegrationTest.java

```diff
@@ -0,0 +1,23 @@
+package com.netflix.eureka2.integration.startup;
+
+import com.netflix.eureka2.server.EurekaWriteServer;
+import org.junit.Test;
+
+/**
+ * Since this test relies on Archaius which has some global settings, we can't have different tests for both read
+ * and write servers under the same JVM. As the tests are testing startup and shutdown behaviour that is common to
+ * both (within AbstractEurekaServer), we just pick one of the two server types (write servers) to test this.
+ *
+ * @author David Liu
+ */
+public class FileBasedServerStartupAndShutdownIntegrationTest extends WriteServerStartupAndShutdownIntegrationTest {
```

Comment:
Why this test is extract in a separate class, and cannot be part of WriteServerStartupAndShutdownIntegrationTest?